ts_scripts/install_utils

```diff
@@ -18,7 +18,13 @@ install_java_deps()
   then
     if [[ "$OSTYPE" == "darwin"* ]]; then
       brew tap AdoptOpenJDK/openjdk
-      brew cask install adoptopenjdk11
+      BREW_VERSION=$(brew --version  2>&1 | head -1 | cut -d' ' -f2)
```

Comment:
NIT (Can be done later): We should remove this script completely after removing the 1 [usage](https://github.com/pytorch/serve/blob/master/binaries/pip/build_wheels.sh). No point in having these when we have migrated everything to python